addons/event_sale/tests/test_event_sale.py

```diff
@@ -31,6 +31,13 @@ def setUp(self):
             'event_id': event.id,
         })
 
+        group_sale_user = self.env.ref('sales_team.group_sale_salesman')
+        self.user_salesperson = self.env['res.users'].create({
+            'name': 'Super Seller',
+            'login': 'salesperson',
+            'groups_id': [(6, 0, [group_sale_user.id])],
```

Comment:
Don't we have to give him base.group_user + sale group ? Don't remember if internal user is implied or not.

Reply:
It seems ok:
https://github.com/odoo/odoo/blob/2f7c03d9ca5809b65dc6eaa778064668d34fcd2e/addons/sales_team/security/sales_team_security.xml#L3-L8